docs/getting-started/operator-quick-start.md

```diff
@@ -0,0 +1,429 @@
+    uid: 7685a5b6-a626-4483-8c20-06b29643a2a8
+  resourceVersion: "8430"
+  uid: 83c5a80d-2dca-46ce-806b-6fdf7bde901f
+spec:
+  type: xdp
+status:
+  conditions:
+  - lastTransitionTime: "2024-12-18T22:27:11Z"
+    message: Successfully loaded bpfProgram
+    reason: bpfmanLoaded
+    status: "True"
+    type: Loaded
+```
+
+### Deploy Namespace Scoped Sample
+
+The namespace scoped samples need a namespace and pods to attach to.
+A yaml has been created that will create a `Namespace`
+([bpfman-operator/hack/namespace_scoped.yaml](https://github.com/bpfman/bpfman-operator/blob/main/hack/namespace_scoped.yaml)).
+The reason for namespace scoped CRDs is to limit an application or user to a namespace.
+To this end, this yaml also creates a `ServiceAccount`, `Role`, `RoleBinding` and `Secret`.
+
+```bash
+cd bpfman-operator
+kubectl apply -f hack/namespace_scoped.yaml 
+  namespace/acme created
+  serviceaccount/test-account created
+  role.rbac.authorization.k8s.io/test-account created
+  rolebinding.rbac.authorization.k8s.io/test-account created
+  secret/test-account-token created
+```
+
+To create a `kubeconfig` file that limits access to the created namespace, use the script 
+[bpfman-operator/hack/namespace_scoped.sh](https://github.com/bpfman/bpfman-operator/blob/main/hack/nginx-deployment.sh).
+The script needs to know the name of the `Cluster`, `Namespace`, `Service Account` and `Secret`.
+The script defaults these fields to what is current in 
+[bpfman-operator/hack/namespace_scoped.yaml](https://github.com/bpfman/bpfman-operator/blob/main/hack/namespace_scoped.yaml).
+However, if a file is passed to the script, it will look for the `Secret` object and attempt to extract the values. 
+This can be used if the names are changed or a different yaml file is used.
+The output of the script is the contents of a `kubeconfig`.
+This can be printed to the console or redirected to a file.
+
+```bash
+./hack/namespace_scoped.sh hack/namespace_scoped.yaml > /tmp/kubeconfig 
+```
+
+To use the `kubeconfig` file, select the session to limit access in and run:
+
+```bash
+export KUBECONFIG=/tmp/kubeconfig
+```
+
+From within this limited access session, a sample `nginx` deployment can be created in the same namespace using
+[bpfman-operator/hack/namespace_scoped.yaml](https://github.com/bpfman/bpfman-operator/blob/main/hack/nginx-deployment.yaml).
+
+```bash
+kubectl apply -f hack/nginx-deployment.yaml
+  deployment.apps/nginx-deployment created
+```
+
+Finally, load any of the namespaced samples from
+[bpfman-operator/config/samples](https://github.com/bpfman/bpfman-operator/tree/main/config/samples).
+They are of the format: `bpfman.io_v1alpha1_*nsprogram.yaml`
+
+```bash
+kubectl apply -f config/samples/bpfman.io_v1alpha1_tc_pass_tcnsprogram.yaml 
+  tcnsprogram.bpfman.io/tc-containers created
+```
+
+If loading of the TC Namespace Program to the selected nodes was successful it will be reported
+back to the user via the `tcNsProgram`'s status field:
```

Comment:
I think it would be better to say how it works in general, and then use TC as an example.

```suggestion
The status for each namespaced program is reported via the \*NsProgram status field and further information
can be seen in the resulting BpfNsProgram CRDs.  As an example, the following commands display the information 
of the TC program loaded in the acme namespace with the command above.
```

Reply:
Done.